app/main/css/themes/dark.css

```diff
@@ -1,22 +1,22 @@
 :root {
   /* Main fonts and colors */
-  --main-background-color: rgba(62, 65, 67, 1);
-  --main-font: system;
-  --main-font-color: white;
+  --main-background-color: rgba(0, 0, 0, 0);
+  --main-font: system, -apple-system, BlinkMacSystemFont;
```

Comment:
Shouldn't this be the other way around? This way the `system` always gets used first and the others only if `systems` does not exist, which should barely ever happen.

Reply:
Here is how I would default a san-serif font: 
--main-font: -apple-system, BlinkMacSystemFont, 'avenir next', avenir, 'helvetica neue', helvetica, ubuntu, roboto, noto, 'segoe ui', arial, sans-serif; 

Or if you want to let the system decide you can use font-family: sans-serif;